src/main/java/net/minestom/server/utils/consumer/EntityConsumer.java

```diff
@@ -1,8 +1,9 @@
 package net.minestom.server.utils.consumer;
 
 import net.minestom.server.entity.Entity;
+import org.jetbrains.annotations.NotNull;
 
 @FunctionalInterface
 public interface EntityConsumer {
```

Comment:
Why does this need to exist? Can't we just use `Consumer<Entity>`?

Reply:
Because it would be impossible to override the method signature to `Consumer<Player>`